### DISCUSSION TITLE
winget version outdated?

### DISCUSSION THREAD

**Discussion by SynapseScribe** [bostrot/wsl2-distro-manager#233] [Q&A] [answered]
i am using wingetui to update apps. currently winget provides Bostrot.WSLManager v1.8.5 which was released back in June. Is the update in winget repo done automatically or you upload it manually?

**Comment by bostrot** [comment]
It is uploaded manually, as I must wait for Microsoft to sign those msix packages.

**Reply by SynapseScribe** [reply]
what about publishing the self signed one (portable zip) instead, maybe under a different package ID such as Bostrot.WSLManger.portable? if winget doesn't allow it, chocolatey should do (as seen similar portable apps in their repo - eg. rbtray app). Portable apps normally go under C:\ProgramData\chocolatey\lib

As an example, I am currently using WingetUI to update my apps, which is quite great and makes use of multiple package managers. However from all the apps I use, your app is the only one outdated and unable to udpate it using wingetui.

**Reply by bostrot** [reply, marked as answer]
Hmm, weird because I know there is an "unofficial" one that gets updated via a script for Chocolatey: https://community.chocolatey.org/packages/wsl2-distro-manager

**Reply by SynapseScribe** [reply]
uh my bad i got confused by the package name i found which is different on winget than on chocolatey. you're right

thank you !!!

**Comment by stamminator** [comment]
Looks like it's out of date again. 
```
Name        Id                 Version Source
----------------------------------------------
WSL Manager Bostrot.WSLManager 1.8.17  winget
```

**Reply by bostrot** [reply]
As you can see in the readme it is generally outdated. I am currently not maintaining this.
